src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -325,7 +325,7 @@ RoutedSavableApp.propTypes = {
   }),
   FormApp: PropTypes.any,
   formConfig: PropTypes.shape({
-    additionalRoutes: PropTypes.object,
+    additionalRoutes: PropTypes.array,
```

Comment:
Does this have implications for other uses of `additionalRoutes`? Why do we need to change to array here?